### DISCUSSION TITLE
薩爾瓦多的喬瓦尼：在農田中辛勤耕耘，等待夢想實現的孩子

### DISCUSSION THREAD

**Discussion by Guagua1234** [1101-GS4719-C/Python-Community-GS4719-C#1140] [General]
https://www.thenewslens.com/feature/2021-worldvision/169517
我覺得薩爾瓦多的童工喬瓦尼的生活和工作狀況非常糟糕。我認為聯合國兒童基金會和國際勞工組織的統計，全球童工人數在過去4年增加了840萬人，達到1.6億人。而在薩爾瓦多，三分之二童工生活在農村地區，主要從事務農或雇工工作。喬瓦尼是其中之一，他7歲就開始幫爺爺在玉米田幫忙，去年開始去其他人田裡工作，希望能幫家裡賺錢。我覺得雖然工作辛苦，但他還是喜歡工作，並且在學校的成績也很好。

**Comment by 110302550** [comment]
這篇文章看了讓我很很難過，有些人為了賺錢，從小就開始工作，我們有了豐衣足食的生活後卻想要更多，真的很不應該

**Comment by woohsin** [comment]
童工問題的存在不僅影響著孩子的身心健康，也限制了他們的學習和成長。這些孩子被迫放棄了接受教育的權利，這對他們的未來發展造成了嚴重的影響。因此，我們需要加強全球合作，通過政策和行動，保護童工的權利，為他們提供更好的教育和生活條件，讓他們能夠擁有一個充滿希望和可能性的未來。
